web/server.js

```diff
@@ -1,34 +1,210 @@
 // modules =================================================
-var express        = require("express");
-var app            = express();
-var bodyParser     = require("body-parser");
+var express		= require("express");
+var app			= express();
+var mongoose	   = require("mongoose");
+var bodyParser	 = require("body-parser");
 var methodOverride = require("method-override");
+var session = require("express-session");
+var passport = require("passport");
+var passportGoogle = require("passport-google-oauth2");
+var GoogleStrategy = passportGoogle.Strategy;
+var passportFacebook = require("passport-facebook");
+var FacebookStrategy = passportFacebook.Strategy;
+var passportGoogleToken = require("passport-google-token");
+var GoogleTokenStrategy = passportGoogleToken.Strategy;
+var passportFacebookToken = require("passport-facebook-token");
+var FacebookTokenStrategy = passportFacebookToken;
 var requireDir = require("require-dir");
 requireDir("./app/libs");
+_ = require("underscore");
 
 // configuration ===========================================
 
+// If you don't have this file, contact Dylan for it.
+var config = require("./config.json");
+
 // config files
 var db = require("./config/db");
-var prodPort = 80;
-var devPort = 8080;
+
+var prodPort = config.ProdPort;
```

Comment:
Ah, so you can specify the production port that you want to use in the config file! Cool!